docker-compose.yml

```diff
@@ -138,7 +138,7 @@ services:
       AWS_SECRET_ACCESS_KEY: minio123
       AWS_DEFAULT_REGION: us-east-1
     volumes:
-      - "./docker/object-storage-init/data/oicr.icgc.test/data:/score-data:ro"
+      - "./docker/object-storage-init/data/score.data/data:/score-data:ro"
```

Comment:
Can you rename the directory this references too? This folder needs to match the volume referenced here.

https://github.com/overture-stack/score/tree/develop/docker/object-storage-init/data/oicr.icgc.test/data